features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplay.kt

```diff
@@ -83,4 +98,14 @@ object SessionReplay {
 
         sessionReplayFeature?.manuallyStopRecording()
     }
+
+    private fun isAlreadyRegistered() =
+        currentRegisteredCore?.get() != null
```

Comment:
> [!IMPORTANT]
> The fact that a core instance is still present doesn't mean it's still active. We need to surface an `isActive()` function on the SDKCore interface to check whether or not the last registered core is still listening and sending events.

Reply:
I've added a check for whether the core is still active